http-server/src/main/java/org/triplea/server/moderator/toolbox/api/key/validation/ApiKeyLookup.java

```diff
@@ -0,0 +1,29 @@
+package org.triplea.server.moderator.toolbox.api.key.validation;
```

Comment:
api/key/validation package has a number of helper (package-protected) classes to implement the overall functionality. To see the high level flow, see the service class: https://github.com/triplea-game/triplea/pull/4872/files#diff-8a0be339eef1a70416907b4c3ef2268e